app/stores/toplevel/ProfileStore.js

```diff
@@ -119,7 +121,9 @@ export default class SettingsStore extends Store {
 
   // ========== Current/Custom Theme ========== //
 
-  @computed get currentTheme(): string {
+  @observable isClassicThemeActive = true;
```

Comment:
I think we want more flexibility related to themes. Probably we want to add this information to the theme itself rather than as a `boolean` here cc @SebastienGllmt